doc/orchestrator.rst

```diff
@@ -166,325 +166,388 @@ A full example of configuring KeyDB to run in SmartSim is shown below
 
-The ``Experiment.start()`` function launches the ``Orchestrators`` for use within the workflow. In other words, the function
-deploys the databases on the allocated compute resources.
+Colocated database:
 
-.. note::
-  Once the multiple ``Orchestrators`` have been launched with ``Experiment.start()``, SmartSim creates separate
-  ``SSDB`` environment variables per orchestrator with the ``db_identifier`` argument suffixed. In this example, the created env vars are
-  ``SSDB_single_shard_db_identifier`` and ``SSDB_clustered_db_identifier``. The environment variables
-  hold the address of the associated database. The two environment variables will be used later in the example
-  to establish a SmartRedis ``Client`` connection with the launched databases.
+.. code-block:: python
 
-Next within the workflow, we send tensors to the launched databases with the ``Client`` SmartRedis API.
+  colo_client = Client(colo_config, logger_name="Model: colo logger")
 
-The SmartRedis ``Client`` object contains functions that manipulate, send, and receive
-data within the database. Each database has a single, dedicated SmartRedis ``Client``.
-Begin by initializing a SmartRedis ``Client`` object per launched database.
+Retrieve Data and Store Using Clients
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To confirm a successful connection to each database, we will retrieve the tensors
+that we plan to store in the python driver script. After retrieving, we
+store both tensors in the colocated database.
+The ``Client.get_tensor()`` method allows
+retrieval of a tensor. It requires the `name` of the tensor assigned
+when sent to the database via ``Client.put_tensor()``.
 
-To create a designated SmartRedis ``Client``, you need to specify the address of the target
-running database. You can easily retrieve this address using the ``get_address()`` function
-provided by the initialized ``Orchestrator`` object.
+.. code-block:: python
 
-.. note::
-  Make sure to set the ``cluster`` argument appropriately based on whether the database is in a clustered configuration or not.
+  val1 = app_single_shard_client.get_tensor("tensor_1")
+  val2 = app_multi_shard_client.get_tensor("tensor_2")
+  app_single_shard_client.log_data(LLInfo, f"The colocated db tensor is: {val1}")
+  app_multi_shard_client.log_data(LLInfo, f"The clustered db tensor is: {val2}")
 
-Here's the code example:
+Later, when you run the experiment the following output will appear in ``tutorial_mode.out``
+located in ``getting-started-multidb/tutorial_model/``.
 
-.. code-block:: python
+.. code-block:: bash
 
-  sr_client_colo = Client(cluster=False, address=colocated_orch.get_address()[0], logger_name="Colo client logger")
-  sr_client_clus = Client(cluster=True, address=clustered_orch.get_address()[0], logger_name="Clus client logger")
+  Model: multi shard logger@00-00-00:The colocated db tensor is: [1 2 3 4]
+  Model: single shard logger@00-00-00:The clustered db tensor is: [5 6 7 8]
```

Comment:
Is the 00-00-00 an anonymized name? If so, maybe replace it with something like "tutorialhost"

Reply:
the 00-00-00 is the time!